### ISSUE TITLE
Tests fail (with Mojolicious::Plugin::AssetPack 0.68?)

### ISSUE THREAD

**Issue by eserte** [dgikiller/mojolicious-plugin-jquery#8]
t/synopsis.t fails, probably only with Mojolicious::Plugin::AssetPack 0.68, but not with earlier versions:
```
...
# production
[AssetPack] Neither readable, nor writeable 'out_dir' directory could be found in static paths /bbbike/perl-5.23.4/lib/site_perl/5.23.4/public. Maybe you forgot to pre-pack the assets? https://metacpan.org/pod/Mojolicious::Plugin::AssetPack::Manual::Cookbook at /opt/perl-5.23.4/lib/site_perl/5.23.4/Mojolicious/Plugin/AssetPack.pm line 178, <DATA> line 2125.
t/synopsis.t ....... 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
...
```

**Comment by dgikiller** [comment]
Thank you for your report Slaven Rezić, this seems to be related to the
AssetPack upgrade as you suggested. I will fix it ASAP.

Il giorno dom 1 nov 2015 alle ore 19:46 Slaven Rezić <
notifications@github.com> ha scritto:

> t/synopsis.t fails, probably only with Mojolicious::Plugin::AssetPack
> 0.68, but not with earlier versions:
>
> ...
> # production
> [AssetPack] Neither readable, nor writeable 'out_dir' directory could be found in static paths /bbbike/perl-5.23.4/lib/site_perl/5.23.4/public. Maybe you forgot to pre-pack the assets? https://metacpan.org/pod/Mojolicious::Plugin::AssetPack::Manual::Cookbook at /opt/perl-5.23.4/lib/site_perl/5.23.4/Mojolicious/Plugin/AssetPack.pm line 178, <DATA> line 2125.
> t/synopsis.t .......
> Dubious, test returned 2 (wstat 512, 0x200)
> No subtests run
> ...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dgikiller/mojolicious-plugin-jquery/issues/8>.
>
-- 

Francesco Ferretti.

**Comment by dgikiller** [comment]
This closes #ca6686d12fa6bc4833cf114588de8b8dfecb24d1
